server/leaderboard/views.py

```diff
@@ -65,3 +65,39 @@ def get_queryset(self):
             )
 
         return queryset.order_by(order_field)
+
+
+class GitHubLeaderboardView(generics.ListAPIView):
+    serializer_class = GitHubStatsSerializer
+    permission_classes = [IsApiKey]
+
+    def get_queryset(self):
+        order_by = self.request.query_params.get("order_by", "commits")
+        time_range = self.request.query_params.get("updated_within", None)
+
+        queryset = GitHubStats.objects.all()
+
+        # time filter
+        if time_range:
+            try:
+                hours = int(time_range)
+                cutoff = timezone.now() - timedelta(hours=hours)
+                queryset = queryset.filter(last_updated__gte=cutoff)
+            except ValueError:
+                raise ValidationError("updated_within must be a valid number of hours")
+
+        # ordering
```

Comment:
lol really? I didn't know ordering_options would be ordering.

Reply:
comments

Reply:
i like to be able to glance quickly at where logical sections start and stop, TBS these aren't super long yet so it isn't doing much

Reply:
you're just trying to pad your comment stat in case I add it to leaderboard 😂